elevate-cpanel

```diff
@@ -2265,7 +2356,6 @@ sub yum_list ( $invalidate_cache = 0 ) {
     while ( my $line = shift @lines ) {
         last if $line && $line =~ m/^Installed Packages/;
     }
-    shift @lines;
```

Comment:
indeed not sure why that `shift` was there... the first line after `Installed Packages` seems a legit package